packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -1186,6 +1186,7 @@ const schema: SchemaType<DbUser> = {
     group: formGroups.deactivate,
   },
 
+  // DEPRECATED
```

Comment:
We can't drop it entirely in one go due to needing to support the old fragments for a bit